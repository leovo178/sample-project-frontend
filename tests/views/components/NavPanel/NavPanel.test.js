import React from 'react'

import { cleanup } from '@testing-library/react'

import { NavPanel } from '@views_components'

import { findDOMNodeOfNavPanel } from './helpers'

describe('<NavPanel />', () => {
	afterEach(() => {
		cleanup()
	})

	it('should match snapshot', () => {
		const { container } = findDOMNodeOfNavPanel(<NavPanel />)

		expect(container).toMatchSnapshot()
	})
})
