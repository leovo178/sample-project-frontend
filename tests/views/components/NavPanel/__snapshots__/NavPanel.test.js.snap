// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavPanel /> should match snapshot 1`] = `
<div>
  <div
    class="MuiGrid-root makeStyles-root-1"
  >
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-tab-2 makeStyles-active-3 MuiButton-textPrimary"
      data-testid="navbaritem-main"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        main
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-tab-2 MuiButton-textPrimary"
      data-testid="navbaritem-user"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        user
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-tab-2 MuiButton-textPrimary"
      data-testid="navbaritem-message"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        message
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
    <button
      class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-tab-2 MuiButton-textPrimary"
      data-testid="navbaritem-file"
      tabindex="0"
      type="button"
    >
      <span
        class="MuiButton-label"
      >
        file
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
  <button
    class="MuiButtonBase-root MuiButton-root MuiButton-text makeStyles-root-139 MuiButton-textPrimary"
    data-testid="logoutbutton"
    tabindex="0"
    type="button"
  >
    <span
      class="MuiButton-label"
    >
      Logout
    </span>
    <span
      class="MuiTouchRipple-root"
    />
  </button>
</div>
`;
